util/pkg/vfs/tests/s3fs_test.go

```diff
@@ -51,9 +53,8 @@ func TestS3RenderTerraform(t *testing.T) {
 			`,
 		},
 	}
-	t.Setenv("S3_ENDPOINT", "foo.s3.amazonaws.com")
-
-	t.Setenv("KOPS_STATE_S3_ACL", "bucket-owner-full-control")
+	viper.Set("S3_ENDPOINT", "foo.s3.amazonaws.com")
+	viper.Set("KOPS_STATE_S3_ACL", "bucket-owner-full-control")
```

Comment:
This is not really required. The problem is mainly that `viper.BindEnv("ENV_VAR")` has to be called before using `viper.GetString("ENV_VAR")`.